packages/optimize/ios/src/RCTAEPOptimize.m

```diff
@@ -0,0 +1,259 @@
+/*
+ Copyright 2021 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+#import "RCTAEPOptimize.h"
+#import <React/RCTEventEmitter.h>
+@import AEPOptimize;
+@import AEPServices;
+@import Foundation;
+
+static NSString * const TAG = @"RCTAEPOptimize";
+
+@implementation RCTAEPOptimize {
+    bool hasListeners;
+}
+
+- (instancetype)init
```

Comment:
There are yellow warning when I run sample app, it seems like is regarding to init.  Happens for Messaging and optimize. Can you take a look?
![optimizeInitWarning](https://user-images.githubusercontent.com/29806547/174730197-531ce134-c678-4556-b5ed-7d751ce21551.png)
![messagingInitWarning](https://user-images.githubusercontent.com/29806547/174730292-f64d8e2e-ca82-446c-bad6-a5d6c2071084.png)

Reply:
I'll take a look!